pycti/entities/opencti_case_incident.py

```diff
@@ -503,26 +495,26 @@ def create(self, **kwargs):
                         "modified": modified,
                         "name": name,
                         "description": description,
-                        "case_type": case_type,
                         "severity": severity,
                         "priority": priority,
-                        "rating": rating,
```

Comment:
no more `rating` ? it's still declared above in line 112

Reply:
No more rating because a case incident cannot create a rating, but can still receive a rating from a feedback. So this is why we query the rating